app/pages/sections/order-items/catalogue-solutions/order-item/flat/template.test.js

```diff
@@ -0,0 +1,207 @@
+import { componentTester } from '../../../../../../test-utils/componentTester';
+
+const setup = {
+  template: {
+    path: 'pages/sections/order-items/catalogue-solutions/order-item/flat/template.njk',
+  },
+};
+
+describe('catalogue-solution order-item - quantity and estimation page', () => {
+  it('should render a backLink', componentTester(setup, (harness) => {
+    const context = {
+      orderId: 'order-1',
+      backLinkText: 'Go back',
+      backLinkHref: '/organisation/order-1/catalogue-solutions/select/solution/price/recipients/date',
+    };
+
+    harness.request(context, ($) => {
+      const backLink = $('[data-test-id="go-back-link"]');
+      expect(backLink.length).toEqual(1);
+      expect(backLink.text().trim()).toEqual('Go back');
+      expect($(backLink).find('a').attr('href'))
+        .toEqual('/organisation/order-1/catalogue-solutions/select/solution/price/recipients/date');
+    });
+  }));
+  it('should render the "Continue" button', componentTester(setup, (harness) => {
```

Comment:
We should add a blank line between the the two tests.

Reply:
added blank line